pymeasure/instruments/wentworthlabs/s200.py

```diff
@@ -0,0 +1,548 @@
+        "Control the y-axis position in microns",
+        validator=strict_range,
+        values=Y_POS_VALID_RANGE,
+        get_process=lambda r:
+        float(r[1]),
+        check_set_errors=True
+    )
+
+    xy_position = Instrument.control(
+        "PSS XY", "GTXY %d,%d",
+        "Control the xy-axis position in microns",
+        validator=multivalue_strict_range,
+        values=X_POS_VALID_RANGE,
+        get_process=lambda r:
+        [float(number) for ind, number in enumerate(r) if ind > 0],
+        check_set_errors=True
+    )
+
+    z_position = Instrument.measurement(
+        "PSGM",
+        "Measure the z-axis position in microns",
+        separator=" ",
+        get_process=lambda r:
+        float(r[1])
+    )
+
+    x_index = Instrument.control(
+        "RXM", "WXM %d",
+        "Control the x index of the chuck in units of 10 microns ",
+        # AWP compatible: Yes
+        validator=strict_range,
+        values=X_Y_INDEX_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RXM ", ""))
+    )
+
+    y_index = Instrument.control(
+        "RYM", "WYM %d",
+        "Control the y index of the chuck in units of 10 microns ",
+        # AWP compatible: Yes
+        validator=strict_range,
+        values=X_Y_INDEX_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RYM ", ""))
+    )
+
+    theta_position = Instrument.control(
+        "PSTH", "GTTH %d",
+        "Control the rotation of the chuck to the Theta-axis position in millidegrees "
+        "specified in the parameter",
+        # AWP compatible: No
+        validator=strict_range,
+        values=[-359999, 359999],
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("PSTH ", ""))
+    )
+
+    z_overtravel = Instrument.control(
+        "RZIM", "WZIM %d",
+        "Control the z-axis overtravel (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_OVERTRAVEL_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RZIM ", ""))
+    )
+
+    z_grosslift = Instrument.control(
+        "RKGM", "WKGM %d",
+        "Control the z-axis gross lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_GROSSLIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKGM ", ""))
+    )
+
+    z_finelift = Instrument.control(
+        "RKFM", "WKFM %d",
+        "Control the z-axis fine lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_FINELIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKFM ", ""))
+    )
+
+    indexing_mode = Instrument.setting(
```

Comment:
To fit style in [writing boolean properties](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments/properties.html?highlight=enabled#boolean-properties), I think this should be `indexing_mode_enabled`